engine/src/main/java/org/terasology/logic/characters/ServerCharacterPredictionSystem.java

```diff
@@ -16,6 +16,7 @@
 
 package org.terasology.logic.characters;
 
+import com.google.api.client.util.Lists;
```

Comment:
Change it on `com.google.common.collect.Lists`

Why is it work for you? It is must be permited by gradle's `implementation`

Reply:
Weird, I just let intellij import that class. It wouldn't work without it. I can change it to "com.google.common.collect.Lists" in another PR, if you want.